i18n/data/zh-CN.json

```diff
@@ -42,5 +42,5 @@
   "UIKit.SelectInput.noOptionsMessageWithInputValue": "无选项",
   "UIKit.SelectInput.noOptionsMessageWithoutInputValue": "无选项",
   "UIKit.SelectInput.placeholder": "选择...",
-  "UIKit.TimeInput.placeholder": "小时：分钟 上午/下午"
```

Comment:
chinese doesn't use am/pm, and when the "value" is displayed and formatted with moment, it's shown in 24 hour time.